config/helpers/generator.py

```diff
@@ -1,15 +1,62 @@
-from config.datamodel import LlamaFarmConfig, Version
+from pathlib import Path
+from typing import Any
 
+try:
```

Comment:
no reason for this to be in a try block. yaml should be a required dependency and always present